common/src/main/resources/Operator.properties

```diff
@@ -145,12 +145,11 @@ WLSKO-0191=Rolling restart of WebLogic domain {0} completed
 WLSKO-0192=Executing make right domain operation, recheck count for server {0} is {1}.
 WLSKO-0193=Waiting for server {0} to start, recheck count is {1}.
 WLSKO-0194=Internal identity initialization step failed with exception {0}.
-WLSKO-0195=The operator encountered a fatal error: ''{0}'' and will not retry automatically. \
+WLSKO-0195=The operator failed after retrying for {0,number,#} minutes. \
+  This time limit may be specified in spec.failureRetryLimitMinutes. \
+  Please resolve the error and then update 'domain.spec.introspectVersion' to force another retry.
+WLSKO-0196=Introspection encountered a fatal error: ''{0}'' and will not retry automatically. \
   Please resolve the error and then update 'domain.spec.introspectVersion' to force a retry.
-WLSKO-0196=Introspection failed: ''{0}'' after {1} attempt(s) and will not retry. \
```

Comment:
Okay, one more question... Now you don't have a duplicated message ID (thanks), but you are deleting the message for WLSK0-0196, but the constant for this message still exists in MessageKeys.java, which I assume means that it's still referenced somewhere.

The opposite is true for WLSK0-0191 -- here the message still exists in this file, but you've deleted the constant that references it.

Reply:
Either solution is fine, but I would like to keep Operator.properties and MessageKeys.java in sync.